src/layouts/Resources.jsx

```diff
@@ -61,79 +112,108 @@ export default class Resources extends Component {
     }
   }
 
-  updateNewPageName = (event) => {
-    event.preventDefault();
-    this.setState({ newPageName: event.target.value });
-  }
-
   categoryModalToggle = () => {
     this.setState((currState) => ({
       categoryModalIsActive: !currState.categoryModalIsActive,
     }));
   }
 
+  settingsToggle = (event) => {
+    const { id } = event.target;
+    const idArray = id.split('-');
+    let resourcePageIndex = idArray[1];
+
+    // User clicked on the "Create New Resource" button
```

Comment:
Again, this dos not work if there is no `resources` folder in the first place. we should keep this in mind for future PRs

Reply:
Logged in #50